src/content/docs/general/licensing.md

```diff
@@ -12,129 +15,64 @@ redirect_from:
   - /identityserver/v7/fundamentals/license_key/  
 ---
 
+Duende products, except for our [open source tools](https://duendesoftware.com/products/opensource),
```

Comment:
should we start making the distinction between open source and FREE open source?